Council branch

prateekcode opened the pull request:
CouncilLayout Layout has been fixed.
I have used Recyclerview to do this.

prateekcode committed:
Pic added

prateekcode committed:
Border added

prateekcode committed:
RecyclerView Adapter Added

prateekcode committed:
Warning Minimized

prateekcode committed:
Warning Minimized

prateekcode committed:
Bug Fixed

sonusourav commented on .idea/misc.xml:
```diff
@@ -5,7 +5,7 @@
     </configurations>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Use JDK_1_8 only.

sonusourav commented on app/src/main/java/com/iitdh/sonusourav/instigo/Council/CouncilActivity.java:
```diff
@@ -58,22 +69,47 @@ protected void onCreate(Bundle savedInstanceState) {
+        recyclerView.setAdapter(recyclerAdapter);
 
 
+
```
Remove this extra line.

sonusourav commented on app/src/main/java/com/iitdh/sonusourav/instigo/Council/CouncilActivity.java:
```diff
@@ -58,22 +69,47 @@ protected void onCreate(Bundle savedInstanceState) {
+        mCouncilImage.add(R.drawable.music_girl);
+        mCouncilImage.add(R.drawable.emergency_call);
+    }
+
```
Remove this line also and similarly in other places.

sonusourav commented on app/src/main/java/com/iitdh/sonusourav/instigo/Council/CouncilRecyclerAdapter.java:
```diff
@@ -0,0 +1,102 @@
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
```
Run Optimise the imports option in android studio.

sonusourav commented on app/src/main/java/com/iitdh/sonusourav/instigo/Council/CouncilRecyclerAdapter.java:
```diff
@@ -0,0 +1,102 @@
+    private ArrayList<Integer> mCouncilImage;
+
+    CouncilRecyclerAdapter(ArrayList<String> mCouncilName, ArrayList<Integer> mCouncilImage) {
+        this.mCouncilName = mCouncilName;
```
Don't use `Hungarian notation`. [Reason](https://stackoverflow.com/questions/111933/why-shouldnt-i-use-hungarian-notation)

sonusourav commented on app/src/main/java/com/iitdh/sonusourav/instigo/Council/CouncilRecyclerAdapter.java:
```diff
@@ -0,0 +1,102 @@
+    @Override
+    public ItemViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+        LayoutInflater inflater = LayoutInflater.from(parent.getContext());
+        View view = inflater.inflate(R.layout.council_single_item, parent, false);
```
These two lines can be made into one. Don't create unnecessary variables.

sonusourav commented on app/src/main/res/layout/council_single_item.xml:
```diff
@@ -0,0 +1,40 @@
+        android:layout_width="match_parent"
+        android:layout_height="35dp"/>
+
+
```
Remove extra lines here also.

sonusourav reviewed (changes requested):
Compress the images before pushing. Use [this](https://www.iloveimg.com/resize-image)

prateekcode committed:
Image Compressed

prateekcode committed:
Used JDK 1_8

prateekcode committed:
Extra Lines Removed

prateekcode committed:
Import Optimized

prateekcode committed:
Hungarian Notation Removed

prateekcode committed:
Unnecessary variables removed

prateekcode commented:
As after your suggestion I fixed the following issues:
1. Compressed Image
2. Used JDK 1_8
3. Removed all extra lines
4. Optimized Imports
5. Not using Hungarian Notation
6. Not using unnecessary variables
